src/pages/cycles/[id].js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import CycleSection from '@components/cycle-section/cycle-section/CycleSection';
+import { Skeleton } from 'primereact/skeleton';
+
+const CyclePage = () => {
+  const { query, isReady } = useRouter();
+
+  if (!isReady)
+    return <Skeleton shape="rectangle" width="100%" height="100%" />;
+
+  const cycleId = query.id;
+
+  if (process.env.NODE_ENV === 'development') {
+    return (
+      <>
+        <CycleSection cycleId={cycleId} />
+      </>
+    );
+  } else {
+    return <p>WIP</p>;
```

Comment:
<img width="137" alt="image" src="https://user-images.githubusercontent.com/42187168/200141101-c7182f32-590a-4918-a2a6-d29af1f5e3c7.png">

Reply:
You merged a dummy graph into main, this is significantly better